docroot/themes/humsci/humsci_basic/patterns/grid/grid.ui_patterns.yml

```diff
@@ -0,0 +1,11 @@
+grid:
+  label: "Grid Layout"
+  description: "Grid layout for content area Views and Paragraphs"
+  fields:
```

Comment:
is it possible to provide `preview` for each field? I know some patterns its more difficult to preview than its worth.

Reply:
yep for sure!

Reply:
good call